src/chargen.cpp

```diff
@@ -2037,6 +2131,33 @@ void create_parse(struct descriptor_data *d, const char *arg)
       priority_menu(d);
     break;
 
+  case CCR_OTAKU_PATH:
+  #define PATH_HELP_STRING "\r\nCyberadepts express their connection to the matrix in a formulaic and rational way. Their complex forms are more potent." \

```

Comment:
What is the gameplay impact of more potent forms vs easier channel/command? Are cyberadepts better at matrix combat than technoshamans or something?

Reply:
ACIFS is a technoshaman's specialty, and operational utilities are a cyberadept's specialty.

Reply:
Got it, is that worth writing in here somehow? I'm sure we're going to have folks who want to play "magic matrix characters" based on flavor without actually reading the otaku rules first.

Reply:
Added it to the help file!